docs/source/notebook_gen_sphinxext.py

```diff
@@ -78,7 +78,8 @@ def generate_rst(app):
     for fname in glob.glob(os.path.join(app.srcdir, notebook_source_dir, '*.ipynb')):
         write_nb(os.path.join(app.srcdir, generated_source_dir), *nb_to_rst(fname))
     with open(os.path.join(app.srcdir, 'examples', 'index.rst'), 'w') as test:
-        test.write('==============\n''MetPy Examples\n''==============\n')
+        test.write('==============\n''MetPy Examples\n''==============\n'
+                   '.. toctree::\n   :glob:\n   :hidden:\n   generated/*\n\n')
```

Comment:
Add another `\n` after `:hidden:`. It seems sphinx wants a full blank line between the `:hidden:` and `generated/*`.